e2e/libs/workload/workload.py

```diff
@@ -116,10 +164,22 @@ def delete_statefulset(name, namespace='default'):
         time.sleep(RETRY_INTERVAL)
     assert deleted
 
-def create_pvc(filepath):
+def create_pvc(volume_type, option):
+    filepath = "./templates/workload/pvc.yaml"
     with open(filepath, 'r') as f:
         namespace = 'default'
         manifest_dict = yaml.safe_load(f)
+        suffix = f"-{volume_type}"
+        if option:
+            suffix += f"-{option}"
+        # correct pvc name
+        manifest_dict['metadata']['name'] += suffix
```

Comment:
Suggest having a until function to generate the name for each resource. It seems the logic here is repeated in other creation functions.

Reply:
Created `get_name_suffix` function for this.